### ISSUE TITLE
preprocess the dataset

### ISSUE THREAD

**Issue by Joseph-LIN-Tech** [OpenImagingLab/TimeLens-XL#4]
Thanks for your dataset. I found the folders in the downloaded data compression package looks messy. 
Is there any scripts to process the data or is there a problem when packaging?

**Comment by seanko29** [comment]
@Joseph-LIN-Tech Did you solve the problem? 
the HQ-EVFI Dataset has too much subfolders and there are no detailed explanation on how to solve them :(

**Comment by Enigmatic-bot-hash** [comment]
在主页有train和test，但是代码需要修改

**Comment by seanko29** [comment]
@Enigmatic-bot-hash yeah I saw that issue too that there are unrelated frames inserted within datasets.
Does that occur only when interp_ratio is set to 4? 
I see in some other ratios.  I didnt test this model in awhile.
